.maintain/weight-template.hbs

```diff
@@ -12,6 +12,7 @@
 #![cfg_attr(rustfmt, rustfmt_skip)]
 #![allow(unused_parens)]
 #![allow(unused_imports)]
+#![allow(non_snake_case)]
```

Comment:
Why?

Reply:
Because in weights there are functions generated with non snake case